### DISCUSSION TITLE
Metaphor Story - useIsomorphicLayoutEffect React hook | Metaphore (SCP)

### DISCUSSION THREAD

**Discussion by giscus** [StreetCommunityProgrammer/metaphore#70] [Metaphor - Discussion]
# Metaphor Story - useIsomorphicLayoutEffect React hook | Metaphore (SCP)

mkubdev telling a story about useIsomorphicLayoutEffect React hook

https://metaphore.vercel.app/stories/javascript/useisomorphiclayouteffect-react-hook

**Comment by darkterminal** [comment]
Write this story while learning something new. That's an awesome catch!!! :rocket:
